Python/Image To Sepia Effect/sepia image convertor.py

```diff
@@ -0,0 +1,24 @@
+# Importing libraries
+import cv2
+import numpy as np
+from PIL import Image
+
+# Taking input from the user
+filename = input("Enter the file name: ")
+img = Image.open("C:/Users/Sharvari/Desktop/Internships/Courses & Events/HackinCodes/Image To Sepia Effect/Image To Sepia Effect/"+ filename)
```

Comment:
Remove this. It's simply redundant as it references your Local Machine Configurations

Reply:
Done